graylog-project-parent/pom.xml

```diff
@@ -416,6 +417,17 @@
                 <scope>provided</scope>
             </dependency>
             <dependency>
+                <groupId>com.gabrielittner.auto.value</groupId>
+                <artifactId>auto-value-extension-util</artifactId>
+                <version>${auto-value-extension-util.version}</version>
```

Comment:
Shouldn't this be marked as `provided` to avoid pulling it into the regular dependencies of the server? Or is this implicitly provided because the auto-value-javabean itself is in scope `provided`?